mona_sdk/validation.py

```diff
@@ -61,17 +66,11 @@ def _is_json_serializable(message):
     return True
 
 
-def validate_mona_single_message(message_event, raise_export_exceptions):
+def validate_mona_single_message(message_event):
     # Check that message_event contains all required fields.
     required_fields = ("message", "contextClass")
 
-    if not is_dict_contains_fields(message_event, required_fields):
-        return handle_export_error(
-            "Messages to export must be of MonaSingleMessage type.",
-            raise_export_exceptions,
-        )
-
-    return True
+    return is_dict_contains_fields(message_event, required_fields)
```

Comment:
did you mean to remove the logging here? any reason for this?

Reply:
Yes, similar to another thread here: to avoid (in _export_batch_inner) passing the whole messages batch to validate_mona_single_message (so it will log the whole batch in case of a failure), I moved handle_export_error outside to the caller function.
@nemo-mona

Reply:
SG